File: apps/demojava/app/src/main/java/com/tradehousemedia/tradecore/java/TheApplication.java
package com.tradehousemedia.tradecore.java;

import android.app.Application;
import com.tradehousemedia.tradecore.core.TradecoreSDK;
import com.tradehousemedia.tradecore.core.utils.TradecoreLogLevel;

public class TheApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        TradecoreSDK.INSTANCE.setLogLevel(TradecoreLogLevel.DEBUG);
        TradecoreSDK.INSTANCE.initialize("tradecore-stack-65", this);
    }

}

File: demo-apps/demokotlin/app/src/main/java/com/tradehousemedia/tradecore/kotlin/TheApplication.kt
package com.tradehousemedia.tradecore.kotlin

import android.app.Application
import com.tradehousemedia.tradecore.core.TradecoreSDK
import com.tradehousemedia.tradecore.core.utils.TradecoreLogLevel

class TheApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        TradecoreSDK.logLevel = TradecoreLogLevel.DEBUG
        TradecoreSDK.initialize("tradecore-stack-69", this)
    }

}
